code/game/objects/items/storage/belt.dm

```diff
@@ -142,8 +142,7 @@
 		/obj/item/healthanalyzer,
 		/obj/item/dnainjector,
 		/obj/item/reagent_containers/dropper,
-		/obj/item/reagent_containers/glass/beaker,
-		/obj/item/reagent_containers/glass/bottle,
+		/obj/item/reagent_containers/glass/,
```

Comment:
everywhere in this file you have examples on how to do it right, and you still fuck it up

Reply:
hey i never said i was smark okay